src/core/readyException.js

```diff
@@ -0,0 +1,13 @@
+define( [
+	"../core"
+], function( jQuery ) {
+
+"use strict";
+
+jQuery.readyException = function( error ) {
```

Comment:
I agree with @markelog here; we shouldn't be swallowing exceptions in a method that doesn't have the promise interface. I see `jQuery.readyException` as a way for people to opt out of the default sensible reporting if they want to use other tools for that, or to hook up other tools. I think the default should log the exception as this PR does.

It seems most of us agree with this strategy.